ert_gui/plottery/plots/distribution.py

```diff
@@ -58,6 +58,11 @@ def _plotDistribution(axes, plot_config, data, label, index, previous_data):
     @type label: Str
     """
 
+    if data.empty:
+        data = pd.Series()
```

Comment:
This is rather strange construct, ie. is this only to set `data.dtype='float64'`? Since when testing it I got this warning:
```bash
DeprecationWarning: The default dtype for empty Series will be 'object' instead of 'float64' in a future version. Specify a dtype explicitly to silence this warning.
```

Reply:
Which pandas version are you running? I have `pandas 0.25.3 `

Reply:
'1.0.3'

Reply:
`0.25.3` is the last version to support Python 2. We should, however, be compatible with whatever is the latest.

Reply:
And the if check is done because we want data to be a pandas.Series when the rest of the plotting code executes, but now that the API changed `data` is a `DataFrame`. The if statement is there to prevent indexing in an empty DataFrame :slightly_smiling_face:

Reply:
I would argue that we should be `0.24` compatible as that is the last version that is installable on the `RHEL 6` machines.

Reply:
I have changed it to explicitly set `dtype=float64`, so now we should be compatible with everything!